doc/01-About.md

```diff
@@ -55,15 +42,41 @@ They can be stored either in a database or in .ini files.
 See [here](07-Preferences.md#preferences) for more information about a user's preferences
 and how to configure their storage type.
 
-## Documentation <a id="about-documentation"></a>
+## Modules
+
+Modules extend Icinga Web 2 with additional functionality. They allow the integration of
+capabilities into existing views and even other modules. Whether it's a graph provider such
```

Comment:
"Whether it's a graph provider such
as [Graphite](https://github.com/Icinga/icingaweb2-module-graphite) or a UI for the Icinga 2
configuration like the [Director](https://github.com/Icinga/icingaweb2-module-director)."

Might want to complete into a full sentence.

Reply:
> Be it a graph provider such as [Graphite](https://github.com/Icinga/icingaweb2-module-graphite) or a UI for the Icinga 2 configuration like the [Director](https://github.com/Icinga/icingaweb2-module-director).

Better?